bot/exts/evergreen/minesweeper.py

```diff
@@ -278,6 +278,13 @@ def reveal_zeros(self, revealed: GameBoard, board: GameBoard, x: int, y: int) ->
             await game.chat_msg.edit(content=new_msg)
         del self.games[ctx.author.id]
 
+    @reveal_command.error
+    @end_command.error
+    async def keyerror_handler(self, ctx: commands.Context, error: typing.Any) -> None:
+        """Handle `KeyError` that is raised in reveal or end command when getting game."""
+        if isinstance(error, KeyError):
+            await ctx.send("Game don't exist.")
```

Comment:
This is an interesting approach that I did not consider, but I like it a lot. There are, however, a few changes needed.
* There are still places in this cog where the look-up is unsafe and fails on a `KeyError`. The `flag_command` is an example. Please review the **entire** cog to ensure that this PR addresses **all** unsafe look-ups.
* I'm a little bit on the edge about blanket-catching *any* `KeyError` that falls out of the commands. While it's probably ok, my personal approach would have been to define a custom `UserNotPlayingError` exception subclass, and handle specifically only that:
```python
if ctx.author.id not in self.games:
    raise UserNotPlayingError
```
* The type annotation for `error` is unnecessarily lax. At the very least, it should be annotated as an `Exception`.